rosette/rbl/rosette/configuration.rbl

```diff
@@ -44,5 +44,10 @@
 (load 'apropos		'silent)
 (load 'space 'silent)
 (load 'primitives 'silent)
-
-(load 'tests/README 'silent)
+(load 'tests/equiv 'silent)
```

Comment:
Do these need to be included given how they are already included here https://github.com/rchain/rchain/blob/dev/rosette/rbl/rosette/tests/run-tests.ros#L69 ?

Reply:
TBH, I don't think any of this stuff is needed, I suspect it was added to the test.sh script naively. Removed it all.